openedx/core/djangoapps/content_libraries/views.py

```diff
@@ -30,7 +31,7 @@
     LibraryBundleLinkUpdateSerializer,
     LibraryXBlockOlxSerializer,
     LibraryXBlockStaticFileSerializer,
-    LibraryXBlockStaticFilesSerializer,
+    LibraryXBlockStaticFilesSerializer, ContentLibraryAddPermissionByEmailSerializer,
```

Comment:
style nit: when breaking imported names across lines, each import should get a line